chronosphere/tfschema/notification_policy.go

```diff
@@ -85,6 +85,20 @@ var NotificationRouteSchema = typeset.Set{
 		"repeat_interval": Duration{
 			Optional: true,
 		},
+		"group_by": typeset.NotNormalized(&schema.Schema{
```

Comment:
remove `typeset` here

Reply:
I don't think this is possible-- since NotificationRoutesSchema expects an ElemField for each of these (which is different from SignalGroupings which just has a Schema expected for each one)

Reply:
Ah sorry, I forgot that typeset elements need this